### PR TITLE
Migrate additional resources

### PR DESCRIPTION
**Pull request by disrani-px**
**What type of PR is this?**
Feature

**What this PR does / why we need it**:
Adds support for migrating additional types of resources

**Does this PR change a user-facing CRD or CLI?**:
No

**Is a release note needed?**:
```release-note
Added support to migrate following resources 
* DaemonSets
* ServiceAccounts (except default)
* ClusterRoles (if used in the namespace)
* ClusterRoleBindings (if used in the namespace)
```

**Does this change need to be cherry-picked to a release branch?**:
No

### CODE REVIEW

File: pkg/migration/controllers/migration.go [hunk 1]
```diff
@@ -631,12 +635,39 @@ func (m *MigrationController) objectToBeMigrated(
 			return false, nil
 		}
 		return true, nil
-	case "Secret":
```
Comment (piyush-nimbalkar) [inline pkg/migration/controllers/migration.go]: Why is secret object removed from migration?

**Comment by piyush-nimbalkar** [comment]
Where do we migrate daemon set?

File: pkg/migration/controllers/migration.go (hunk 1 above)
Comment (disrani-px) [reply pkg/migration/controllers/migration.go]: It's not removed. This function just returns whether an object should be migrated. Previously we were not returning secrets for secret accounts. By removing this it returns `true,nil` below for all secrets.

**Comment by disrani-px** [comment]
@piyush-nimbalkar The check is in `resourceToBeMigrated()` that now returns true for DaemonSets. `objectToBeMigrated()` wil always return true for all DaemonSets

**Pull request merged**